.vscode/settings.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
This is IDE-specific and does not belong into version control. Please also squash your commits as we generally aim for a helpful commit history and do not squash automatically.